frontends/qt/BitBox.pro

```diff
@@ -56,7 +56,8 @@ SOURCES += \
         main.cpp \
         filedialog.cpp
 
-HEADERS += libserver.h webclass.h filedialog.h
+# server/libserver.h is generated by building server.go, see the Makefiles in server/.
+HEADERS += server/libserver.h webclass.h filedialog.h
```

Comment:
@Beerosagos the file is not committed - not sure if we should commit it or not :thinking: if committed, upgrading Go might change the file and we'd need to make a PR to update it.

Reply:
Yeah I agree, I don't see the advantage of committing it